docs/data/material/getting-started/templates/checkout/Info.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+
+const products = [
+  {
+    name: 'Professional plan',
+    desc: 'Monthly subscription',
+    price: '$15.00',
+  },
+  {
+    name: 'Dedicated support',
+    desc: 'Included in Professional plan',
+    price: 'Free',
+  },
+  {
+    name: 'Hardware',
+    desc: 'Devices needed for development',
+    price: '$69.99',
+  },
+  {
+    name: 'Landing page template',
+    desc: 'License',
+    price: '$49.99',
+  },
+];
+
+export default function Info() {
+  return (
+    <React.Fragment>
+      <Typography variant="subtitle2" color="text.secondary" sx={{ mt: 10 }}>
```

Comment:
```suggestion
      <Typography variant="subtitle2" color="text.secondary">
```

Would try to rely on the parent component, with gap, for the spacing as much as possible